tests/test_freeze.py

```diff
@@ -66,12 +66,21 @@ async def test_freeze_fix_depends(
     assert dep2_metadata["imports"] == toplevel[2]
 
 
-def test_freeze_lockfile_compat(selenium_standalone_micropip, wheel_catalog, tmp_path):
+@pytest.mark.parametrize(
+    ("name", "depends"),
+    [
+        ["pytest", {"attrs", "iniconfig", "packaging", "pluggy"}],
+        ["snowballstemmer", set()],
+    ],
+)
+def test_freeze_lockfile_compat(
```

Comment:
Ran this here: https://github.com/bollwyvl/micropip/actions/runs/12726352585

Reply:
Here's a run with just the test vs main(-ish):

https://github.com/bollwyvl/micropip/actions/runs/12726465651/job/35474893463#step:7:315